spec/box_spec.cr

```diff
@@ -1,19 +1,37 @@
 require "./spec_helper"
 
-describe "Sequences" do
-  it "increases a value every time it's called" do
-    BaseBox::SEQUENCES["name"] = 0
+describe Avram::Box do
+  describe "Sequences" do
+    it "increases a value every time it's called" do
+      BaseBox::SEQUENCES["name"] = 0
 
-    tag1 = TagBox.create
-    tag2 = TagBox.create
+      tag1 = TagBox.create
+      tag2 = TagBox.create
 
-    tag1.name.should eq("name-1")
-    tag2.name.should eq("name-2")
+      tag1.name.should eq("name-1")
+      tag2.name.should eq("name-2")
+    end
+
+    it "can be overridden" do
+      tag = TagBox.create(&.name("not-a-sequence"))
+
+      tag.name.should eq("not-a-sequence")
+    end
   end
 
-  it "can be overridden" do
-    tag = TagBox.create(&.name("not-a-sequence"))
+  describe "create_pair" do
+    it "creates 2 tags" do
+      tags = TagBox.create_pair
+      tags.size.should eq 2
+      tags.class.name.should eq "Array(Tag)"
+    end
 
-    tag.name.should eq("not-a-sequence")
+    it "yields the block to both boxes" do
+      tags = TagBox.create_pair do |box|
+        box.name(box.sequence("new-tag"))
```

Comment:
I wonder if  UserBox might be better here so you could also do `box.age(30)` and check that both records have 30?

Reply:
![why not both](https://i.imgur.com/DBcUtoo.png "Why not both?")